packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -170,13 +187,19 @@ class _AddNewProductPageState extends State<AddNewProductPage> {
     return Padding(
       padding: _ROW_PADDING_TOP,
       child: Row(
-        mainAxisAlignment: MainAxisAlignment.start,
+        mainAxisAlignment: MainAxisAlignment.center,
         children: <Widget>[
           SizedBox(height: 50, child: Image.file(image, fit: BoxFit.cover)),
           Expanded(
-              child: Center(
-                  child: Text(_getPhotoUploadedLabelText(context, imageType),
-                      style: Theme.of(context).textTheme.bodyText1))),
+            child: Center(
+              child: Text(_getAddPhotoButtonText(context, imageType),
+                  style: Theme.of(context).textTheme.bodyText1),
```

Comment:
You're calling Theme.of() twice, please store it in a variable, to prevent multiple calls